examples/js/GPUComputationRenderer.js

```diff
@@ -99,28 +103,41 @@
 
 function GPUComputationRenderer( sizeX, sizeY, renderer ) {
 
-	this.variables = [];
+	this.renderer = renderer;
 
-	this.currentTextureIndex = 0;
+	this.variables = [];
 
+	// setup the scene for rendering
 	var scene = new THREE.Scene();
 
 	var camera = new THREE.Camera();
 	camera.position.z = 1;
 
+	// setup the initial shader for the mesh
 	var passThruUniforms = {
 		texture: { value: null }
 	};
 
-	var passThruShader = createShaderMaterial( getPassThroughFragmentShader(), passThruUniforms );
+	var passThruShader = createShaderMaterial( getPassThroughFragmentShader(sizeX, sizeY), passThruUniforms );
 
 	var mesh = new THREE.Mesh( new THREE.PlaneBufferGeometry( 2, 2 ), passThruShader );
 	scene.add( mesh );
 
+	this.magFilter = THREE.NearestFilter;
+	this.minFilter = THREE.NearestFilter;
+
+	if ( renderer.extensions.get('OES_texture_float_linear') ) {
```

Comment:
It is not desirable to have the filter set to linear by default. For example in the birds demo that will mix the data from neighbour birds. In the protoplanets example the particles will mix their states, and so on.